cores/fancontrol/fancontrol.core

```diff
@@ -0,0 +1,10 @@
+CAPI=1
+[main]
+name = wallento:boards:vcu108-fancontrol
+description = "fan control for VCU108 board"
+
+[fileset rtl_files]
+files =
+  rtl/verilog/fancontrol.sv
+file_type = verilogSource
```

Comment:
please make this `systemVerilogSource` (required by some tools to parse files as SV).

Reply:
done